src/components/simpleareachart.js

```diff
@@ -19,28 +19,37 @@ export default function SimpleAreaChart ({ emissionsData, title }) {
   const YearDataKey = 'year'
   const EmissionsDataKey = 'hist'
   const ProjectionDataKey = 'projection'
+  const MissingDataKey = 'missingData'
 
   const annualHistoricEmissions = emissionsData.map((item) => {
     var data = { [YearDataKey]: item.year, [EmissionsDataKey]: 0 }
 
-    data[EmissionsDataKey] = Math.round(100*Object.entries(item)
+    data[EmissionsDataKey] = Math.round(100 * Object.entries(item)
       .filter(([key, _val]) => key !== YearDataKey)
-      .reduce((acc, [_key, val]) => acc + val, 0)) /100
+      .reduce((acc, [_key, val]) => acc + val, 0)) / 100
 
     return data
   })
-
   const currYear = new Date().getFullYear()
   const yearsLeft = 2050 - currYear
-  const reduceFrom
-    = annualHistoricEmissions[annualHistoricEmissions.length - 1][EmissionsDataKey]
+  const reduceFrom = annualHistoricEmissions[annualHistoricEmissions.length - 1][EmissionsDataKey]
+  const lastYear = annualHistoricEmissions.slice(-1)[0].year
   var projection = []
+  var missing = []
+
+  for (let step = 0; step < (currYear - lastYear - 1); step++) {
```

Comment:
Can you document this code so folks know why it's doing what it does?

Reply:
Done.